requirements-dev.txt

```diff
@@ -2,4 +2,4 @@
 mock
 nose
 coverage
-moto
+git+git://github.com/spulec/moto.git@5324638573e18e51b895b39ec0d32ac2133a37d0#egg=moto
```

Comment:
It should work indefinitely, as it's just pulling from a specific commit on that repository.  That said, once Moto increments their pip package to the next version it might be cleaner to change the requirement back to using the latest package available on pip.